mycalendar/src/webapp/scripts/mycalendar.js

```diff
@@ -0,0 +1,263 @@
+(function($) {
+
+    var currentSiteId;
+    var timeZone;
+    var eventsList = [];
+
+    $(document).ready(function () {
+
+        moment.locale(sakai.locale.userLocale);
+        
+        // data
+        currentSiteId = $(".mycalendar-data").attr("data-siteid");
+        timeZone = $(".mycalendar-data").attr("data-tz");
+        
+        //get all events for the site for this month
+        var todaysDate = moment().format('YYYY-MM-DD');
+        eventList = getEvents(todaysDate);
+
+        var init = function () {
+        
+                $(".mycalendar").datepicker({
```

Comment:
If you are just picking dates and not date+times, this one might be fine. have you tested in european locales? Does the date picker use the correct format for the locale? IF yes, let's proceed.